contributing/DEVELOPMENT.md

```
@@ -0,0 +1,373 @@
+        print(f"Reading partition: {partition.value if hasattr(partition, 'value') else partition}")
+        print(f"Options: {self.options}")
+
+        # Your reading logic
+        for row in data:
+            print(f"Yielding: {row}")
+            yield row
+```
+
+### Common Issues
+
+1. **Serialization errors**: Ensure all class attributes are pickle-able
```

markdownlint-cli2 (0.18.1)

Strong style
Expected: underscore; Actual: asterisk

(MD050, strong-style)